NuRadioReco/detector/response.py

```diff
@@ -127,9 +128,15 @@ def __init__(self, frequency, y, y_unit, time_delay=0, weight=1,
         self.__phases = [interpolate.interp1d(
             self.__frequency, y_phase, kind="linear", bounds_error=False, fill_value=0)]
 
+        if weight not in [-1, 1]:
+            err = f"Only a response weight of [-1, 1] is allowed (value is {weight})."
+            self.logger.error(err)
+            raise ValueError(err)
```

Comment:
I guess you knew (and I guessed, and then tested) that Python is smart enough that this works for floats too : ) 
(not that it would have been an issue if not, just should have raised a more helpful TypeError in that case).

Reply:
Of course I did 😇